people.md

```diff
@@ -124,6 +124,13 @@ members:
     url: http://www.imperial.ac.uk/people/zulma.cucunuba
     github: https://github.com/zulmac
     twitter: https://twitter.com/Zulma.Cucunuba
+  - name: Flavio Finger
+    img: /img/people/flavio-finger.jpg
+    desc: Research Fellow in real-time epidemiological analysis. London School of Hygene and Tropical Medicine, UK.
```

Comment:
This has changed since. Can you put "Epidemiologist & Outbreak Analyst at Epicentre, Paris, France"

Reply:
Done